atomix/storage/src/test/java/io/atomix/storage/journal/AbstractJournalTest.java

```diff
@@ -91,7 +91,7 @@ public static Collection primeNumbers() {
         .withNamespace(NAMESPACE)
         .withStorageLevel(storageLevel())
         .withMaxSegmentSize(maxSegmentSize)
-        .withMaxEntrySize(64)
+        .withMaxEntrySize(48)
```

Comment:
I'm thinking if it would be good to test with multiple max entry sizes to find other potential bugs in future.

Reply:
Are you fine in doing this in a separate issue? Then I would create an issue to improve in the journal tests.

Reply:
Yes :+1:

Reply:
https://github.com/zeebe-io/zeebe/issues/4407